Porting to most played minecraft versions

JayJay1989 opened the pull request:
Ported to most played Minecraft versions

- Forge 1.10.2
- Forge 1.11.2
- Forge 1.12.2

JayJay1989 commented:
yes i have, they're on my GH: [release](https://github.com/JayJay1989/gamesense-sdk/releases)

OneGaming36 commented:
I keep receiving an error right after the start screen where it loads the plugins.

`The game crashed whilst there was a severe problem during mod loading that has caused the game to fail
The game crashed whilst there was a severe problem during mod loading that has caused the game to fail
Error: net.minecraftforge.fml.common.LoaderExceptionModCrash: Caught exception from GameSense Mod (gamesense)`

In the mods-folder there are only your GameSense1.11-forge1.12.2.jar file (I'm using the same forge-version) and the json-20140107.jar file.

Am I doing something wrong?

JayJay1989 commented:
im going to check myself. I think you require forge-1.12.2-14.23.1.2555, not sure

JayJay1989 commented:
only the Gamesense mod is needed. I updated the JSON library: [link](https://github.com/JayJay1989/gamesense-sdk/blob/master/examples/minecraftforge1.12.2/jars/json-20171018.jar)

JayJay1989 commented:
new link to releases: [link](https://github.com/JayJay1989/gamesense-sdk/releases/tag/master)

OneGaming36 commented:
This one works fine, thank you very much. 😄

JayJay1989 commented:
Last change (only for 1.12.2): 
- Add json.org library into the mod itself, no need for the jsonxxxxx.jar file in the mods folder of your game
- Changed some build mechanics
- Added steelseries logo to the mod, viewable under mods in the main menu of the game

download links for the mod: [link](https://github.com/JayJay1989/gamesense-sdk/releases/tag/1.12.2)

PeterGamerrr commented:
Make a 1.8.9 version becouse the biggest mc pvp server in the world uses 1.8 pvp so a lot of players play on 1.8.9.
it would be so awsome if you made it

JayJay1989 commented:
I'll do it later today. I let you know when its ready ;)

JayJay1989 commented:
@PeterGamerrr here you go [Latest releases](https://github.com/JayJay1989/gamesense-sdk/releases/tag/master)
Note: remove your "json.xxxx.jar" in your mods, this mod includes this library
